_justifications/example.md.template

```diff
@@ -0,0 +1,40 @@
+---
+title: Justification title
+---
+
+A brief justification description.
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor
+incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis
+nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu
+fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in
+culpa qui officia deserunt mollit anim id est laborum.
+
+## Pipeline units
+
+ * TensorFlowAVX2Steps
+
+## Recommendation types
```

Comment:
do we have a piece of doc describing what a recommendation type is?

Reply:
We can create it.